Fsxac/m3 d (#62)
Capstone-Skynet/Capstone-Skynet.github.io: docs/design/3_2_multirotor.tex

The bare-minimum multirotor remote-piloted aerial system (RPAS) or informally referred as ``drone'', consists of a propulsion system, flight controller, speed controllers, battery/fuel, and sensor systems.

\subsubsection{Multirotor Configuration}

> A multirotor RPAS (MRPAS) has several common configurations: a tricopter, a quadcopter, and a hexacopter such as seen in figure \ref{fig:mrpas-config} below.

Reviewer: ```suggestion
A multirotor RPAS (MRPAS) has several common configurations: a tricopter, a quadcopter, and a hexacopter (as seen in Figure \ref{fig:mrpas-config}).
```

\begin{figure}[H]
    \centering
    \includegraphics[width=0.85\textwidth]{img/testrigcad1.png}
    \caption{Team 109 Testing Rig Prototype using RPAS Generously Provided by Yiyi Yan and UBC \textit{Unmanned Aerial Systems}}
    \label{fig:testrigcad1}
\end{figure}

> The bare-minimum multirotor remote-piloted aerial system (RPAS) or informally referred as ``drone'', consists of a propulsion system, flight controller, speed controllers, battery, and sensor systems. Our testing rig is shown  in Figure~\ref{fig:testrigcad1}

Reviewer 2: ```suggestion
The bare-minimum multirotor remote-piloted aerial system (RPAS) or informally referred to as ``drone'', consists of a propulsion system, flight controller, speed controllers, battery, and sensor systems. Our testing rig is shown  in Figure~\ref{fig:testrigcad1}
```

    \label{fig:rpas_configs}
\end{figure}

\textbf{Tricopter:} Tricopters requires fewer motors and speed controllers, implying less components and power draw required to sustain flight. The angle between each motor arm is also wider at 120 degrees and thus reduces obstruction from camera views. 

\begin{figure}[h]
    \centering
    \includegraphics[scale=0.4]{img/drone_yconfigt}
    \caption{Tricopter Y-configuration}
    \label{fig:tricopter-y-t}
\end{figure}

> However, systems with odd-number of actuatators are inherently unstable due to the asymmetry of the motor torque as shown in figure \ref{fig:tricopter-y-t}. Therefore, this design require an additional servo motor (consists of a DC motor and a gear box) to precisely control the tail motor angle of tilt to counter the excess torque --- similar to the tail of a helicopter.

Reviewer: ```suggestion
However, systems with an odd-number of actuators are inherently unstable due to an asymmetric motor torque, as shown in Figure \ref{fig:tricopter-y-t}. Therefore, this design requires an additional servo motor (consisting of a DC motor and a gear box) to precisely control the tail motor angle of tilt to counter the excess torque --- similar to the tail of a helicopter.
```

> We choose to not pursue this configuration as the tricopter configuration provides agility with require increased control complexity as a trade off. This compromise is not ideal as we emphasize on flight stability.

Reviewer: This sentence is funky, you might want to reword it.

\textbf{Quadcopter X:}

> The quadcopter is the most common configuration in consumer multirotor products. The quadcopter is more stable than a tricopter because a quadcopter configuration utilizes four motors as seen in Figure \ref{fig:quadcopter-x-t}. Two of the motors spin clockwise and the other two spin counter-clockwise and effectively cancelling each other’s undesired torque. Applying the same power into each motor allows the MRPAS to hover in place. We can perform 6 degree-of-freedom (DOF) movements by changing a combination of differential thrust to each motor (Figure \ref{fig:rpas_6dof}).

Reviewer: ```suggestion
The quadcopter is the most common configuration in consumer multirotor products. The quadcopter is more stable than a tricopter as it utilizes four motors, as seen in Figure \ref{fig:quadcopter-x-t}. Two of the motors spin clockwise and the other two spin counter-clockwise -- effectively cancelling each other’s undesired torque. Applying the same power into each motor allows the MRPAS to hover in place. We can perform 6 degree-of-freedom (DOF) movements by applying a combination of differential thrusts to each motor (Figure \ref{fig:rpas_6dof}).
```

    \end{subfigure}
    
    \caption{Multirotor RPAS Configurations}
    \label{fig:rpas-config}
\end{figure}

\textbf{Tricopter:} Tricopters requires fewer motors and speed controllers, implying less components and power draw required to sustain flight. The angle between each motor arm is also wider at 120 degrees and thus reduces obstruction from camera views. 

\begin{figure}[h]
    \centering
    \includegraphics[scale=0.4]{img/drone_yconfigt}
    \caption{Tricopter Y-configuration}
    \label{fig:tricopter-y-t}
\end{figure}

> However, systems with an odd-number of actuators are inherently unstable due to the asymmetry of the motor torque as shown in Figure~\ref{fig:tricopter-y-t}. Therefore, this design requires an additional servo motor (consists of a DC motor and a gear box) to precisely control the tail motor angle of tilt to counter the excess torque --- similar to the tail of a helicopter.

Reviewer 2: ```suggestion
However, systems with an odd-number of actuators are inherently unstable due to the asymmetry of the motor torque as shown in Figure~\ref{fig:tricopter-y-t}. Therefore, this design requires an additional servo motor (consists of a DC motor and a gearbox) to precisely control the tail motor angle of tilt to counter the excess torque --- similar to the tail of a helicopter.
```

    \label{fig:tricopter-y-t}
\end{figure}

> However, systems with an odd-number of actuators are inherently unstable due to the asymmetry of the motor torque as shown in Figure~\ref{fig:tricopter-y-t}. Therefore, this design require an additional servo motor (consists of a DC motor and a gear box) to precisely control the tail motor angle of tilt to counter the excess torque --- similar to the tail of a helicopter.

Reviewer 3: ```suggestion
However, systems with an odd-number of actuators are inherently unstable due to the asymmetry of the motor torque as shown in Figure~\ref{fig:tricopter-y-t}. Therefore, this design requires an additional servo motor (consists of a DC motor and a gear box) to precisely control the tail motor angle of tilt to counter the excess torque --- similar to the tail of a helicopter.
```

    \label{fig:rpas_6dof}
\end{figure}

\textbf{Hexacopter X:}

> The hexacopter has all the stability benefits of a quadcopter as well as extra redundancy. The MRPAS could still land safely even up to dual motor failures. Since the hexacopter uses two more motors than the quadcopter configuration, naturally, the RPAS of this configuration is generally bigger and can lift more payload. Of course, with the added components, it is also much more expensive to build, operate, and maintain. Furthermore, hexacopter RPAS's extended size and mass can cause more damage upon a crash.

Reviewer: ```suggestion
The hexacopter has all the stability benefits of a quadcopter as well as extra redundancy. The MRPAS could still land safely even if it experienced up to two simultaneous motor failures. Since the hexacopter uses two more motors than the quadcopter configuration, naturally, the RPAS of this configuration is generally bigger and can lift more payload. Of course, with the added components, it is also much more expensive to build, operate, and maintain. Furthermore, hexacopter RPAS's extended size and mass can cause more damage upon a crash.
```

We choose to the quadcopter configuration. 
The payload is not too heavy -- our target maximum payload weight is less than 500 g (\textbf{NF.DR.1}). The quadcopter configuration is also the most balanced in terms of the trade-off between cost and reliability. 

\subsubsection{Air Frame}\label{section:air-frame}

> RPAS air frame sizes are commonly refered by their \textit{motor-to-motor-diagonal-span} (MMDS). The MMDS measures from a pair of motors that are the most far apart. 

Reviewer: ```suggestion
RPAS air frame sizes are commonly referred to by their \textit{motor-to-motor-diagonal-span} (MMDS). The MMDS measures the distance between a pair of motors which are the most far apart on the RPAS. 
```

> The most common frame available for sale comes in 450 mm MMDS variants. 350 mm frames are also common amongst consumer products like the DJI Phantom series\cite{dji-phantom-3-specs} as seen in Figure~\ref{fig:djiphantom}. The frame size of 350 mm will constrain our ability to mount larger hardware --- posing a signifcant risk to integrating computing platform hardware. Additionally The 350 mm size also limits the propeller maximum size. The largest quadcopter configuration has an MMDS of 650 mm to 1000 mm and allow for extremely large payload capacity such as the one shown in Figure~\ref{fig:djis1000}. However, these are more commonly used for industrial or military applications.

Reviewer: ```suggestion
The most common frame available for sale comes in 450 mm MMDS variants. 350 mm frames are also common amongst consumer products like the DJI Phantom series\cite{dji-phantom-3-specs}, as seen in Figure~\ref{fig:djiphantom}. The frame size of 350 mm would constrain our ability to mount larger hardware --- posing a significant risk to integrating computing platform hardware. Additionally the 350 mm size also limits the maximum size of the propeller. The largest quadcopter configurations have an MMDS of 650 mm to 1000 mm and allow for extremely large payload capacities such as the one shown in Figure~\ref{fig:djis1000}. However, these are more commonly used for industrial or military applications.
```

        \caption{Generic DIY frame kit (450 mm)}
    \end{subfigure}
    ~
    \begin{subfigure}[b]{0.33\textwidth}
        \centering
        \includegraphics[width=\textwidth]{img/jmt560}
        \caption{JMT X4 Carbon Fiber Frame (560 mm)}
        \label{fig:jmtx4}
    \end{subfigure}
    ~
    \begin{subfigure}[b]{0.33\textwidth}
        \centering
        \includegraphics[width=\textwidth]{img/djis1000}
        \caption{DJI Spreadwings S1000 (1000 mm)}
        \label{fig:djis1000}
    \end{subfigure}
    
    \caption{Air Frame Options with Various Size Tiers }
\end{figure}

> Our ideal option is a X4 560 mm MMDS carbon fibre frame from JMT, as shown in Figure~\ref{fig:jmtx4}. The frame is carbon fiber to enhance structural rigidity while keeping the system extremely lightweight. While the frame also features foldable arms and landing gear for portability, we consider replacing the hinges with static parts to reduce dead weight in order to achieve a longer flight time, as part of our non-functional requirements (\textbf{NF.DR.7}). The X4 560 mm frame is also wide enough to support propollers up to 14 inches in diameter. See Section~\ref{section:propsys} for reasons why a larger propeller is generally better.

Reviewer: ```suggestion
Our ideal configuration is a X4 560 mm MMDS carbon fibre frame from JMT, as shown in Figure~\ref{fig:jmtx4}. The frame is carbon fiber to enhance structural rigidity while keeping the system extremely lightweight. While the frame also features foldable arms and landing gears for portability, we would consider replacing the hinges with static parts to reduce dead weight in order to achieve a longer flight time, as part of our non-functional requirements (\textbf{NF.DR.7}). The X4 560 mm frame is also wide enough to support propellers up to 14 inches in diameter. 
```

> For our testing rig (which is also our back-up drone), we are using the \textit{Flamewheel F450} frame with 450 mm MMDS. This design has many clones and variants and thus extremely cheap to purchase and replace on the internet. The frame has a adequate balance between portability and capability. However, the 450 mm frames can only support at most 12 inch propellers.

Reviewer: ```suggestion
For our testing rig (which is also our back-up drone), we are using the \textit{Flamewheel F450} frame with 450 mm MMDS. This design has many clones and variants and is thus extremely cheap to purchase and replace on the internet. The frame provides an adequate trade-off between portability and capability. However, the 450 mm frames can only support 12 inch propellers at most.
```

\subsubsection{Propulsion System}\label{section:propsys}

The propulsion system covers the majority of the RPAS parts list and determines the flight performance of the RPAS. 

\paragraph{Motor Type}

\underline{\textit{Brushed or brushless?}}
The MRPAS is electrically powered with a battery (DC) and we have two options for motors: DC brushed  motors and DC brushless motors. Brushed motors use mechanical brushes in contact with rotor’s commutator to switch polarity to sustain a constant rotation, whereas the brushless motors use electronic controllers to switch the polarity in the rotor electrically. 

For flight applications, we chose the brushless DC (BLDC) motors for their extended usable life time. BLDC motors are also much more efficient than brushed motors because as they do not mechanically swap the polarity in the motor, which results in excess friction, heat, and potential sparking. 

\underline{\textit{Inrunner or outrunners?}}

> BLDC motors are further broken down into two types: \textit{inrunner} and \textit{outrunner}. Inrunner motors have the rotor on the inside and the outrunner motors have the rotor on the outside.  As a result, the inrunners spins with a relatively lightweight rotor, which means inrunner motors typically spin fast but have low torque. In order to obtain useful torque from the motor, we need to design and install a separate gearbox to convert the mechanical energies\cite{invsoutrunner}. The outrunner has a more massive rotor and so the spin-up time due to the additional inertia, but we can consider this as a benefit: the rotating mass is further from the center of rotation, thus the angular momentum helps stablizing the rotor and reduce vibrations --- which is a major loss of efficiency. For a MRPAS typically mainly operating in stabalized flight and hover, this ``compromise'' is acceptable. Lastly, outrunners technologies are more modern and more serviceable than inrunner motors\cite{invsoutrunner}. For the reasons above, we choose outrunner motors.

Reviewer: ```suggestion
BLDC motors are further broken down into two types: \textit{inrunner} and \textit{outrunner}. Inrunner motors have the rotor on the inside and outrunner motors have the rotor on the outside.  As a result, inrunners spin with a relatively lightweight rotor, implying that they typically spin quickly but have a low torque. In order to obtain a useful level of torque from the motor, we would need to design and install a separate gearbox to convert the mechanical energy\cite{invsoutrunner}. The outrunner has a more massive rotor and thus a comparatively increased spin-up time due to the additional inertia, but we can consider this as a benefit: the rotating mass is further from the center of rotation and thus the angular momentum helps stabilize the rotor and reduce vibrations --- a major loss of efficiency. For a MRPAS typically operating in stabilized flight and hover, this ``compromise'' is acceptable. Finally, outrunners are ultimately more modern and more serviceable than inrunner motors\cite{invsoutrunner}. For the reasons above, we chose outrunner motors.
```

Reviewer: BTW, is inrunner/outrunner one word or two?

For our testing rig (which is also our back-up drone), we are using the \textit{Flamewheel F450} frame with 450 mm MMDS. This design has many clones and variants and thus extremely cheap to purchase and replace on the internet. The frame has a adequate balance between portability and capability. However, the 450 mm frames can only support at most 12 inch propellers.

\subsubsection{Propulsion System}\label{section:propsys}

The propulsion system covers the majority of the RPAS parts list and determines the flight performance of the RPAS. 

\paragraph{Motor Type}

\underline{\textit{Brushed or brushless?}}

> The MRPAS is electrically powered with a battery (DC) and we have two options for motors: DC brushed  motors and DC brushless motors. Brushed motors use mechanical brushes in contact with rotor’s commutator to switch polarity to sustain a constant rotation, whereas the brushless motors use electronic controllers to switch the polarity in the rotor electrically. 

Reviewer 2: ```suggestion
The MRPAS is electrically powered by a DC battery and we have two options for motors: DC brushed motors and DC brushless motors. Brushed motors use mechanical brushes in contact with the rotor’s commutator to switch polarity to sustain a constant rotation, whereas the brushless motors use electronic controllers to switch the polarity in the rotor. 
```

> For flight applications, we chose the brushless DC (BLDC) motors for their extended usable life time. BLDC motors are also much more efficient than brushed motors because as they do not mechanically swap the polarity in the motor, which results in excess friction, heat, and potential sparking. 

Reviewer 2: ```suggestion
For flight applications, we chose the brushless DC (BLDC) motors for their extended usable lifetime. BLDC motors are also much more efficient than brushed motors because as they do not mechanically swap the polarity in the motor, which results in excess friction, heat, and potential sparking. 
```

\underline{\textit{Inrunner or outrunners?}}

> BLDC motors are further broken down into two types: \textit{inrunner} and \textit{outrunner}. Inrunner motors have the rotor on the inside and the outrunner motors have the rotor on the outside.  As a result, the inrunners spins with a relatively lightweight rotor, which means inrunner motors typically spin fast but have low torque (i.e. inrunner motors have high KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). In order to obtain useful torque from the motor, we need to design and install a separate gearbox to convert the mechanical energies\cite{invsoutrunner}. The outrunner has a more massive rotor and so the spin-up time due to the additional inertia, but we can consider this as a benefit: the rotating mass is further from the center of rotation, thus the angular momentum helps stablizing the rotor and reduce vibrations --- which is a major loss of efficiency. For a MRPAS typically mainly operating in stabalized flight and hover, this ``compromise'' is acceptable. Lastly, outrunners technologies are more modern and more serviceable than inrunner motors\cite{invsoutrunner}. For the reasons above, we choose outrunner motors.

Reviewer 2: ```suggestion
BLDC motors are further broken down into two types: \textit{inrunner} and \textit{outrunner}. Inrunner motors have the rotor on the inside and the outrunner motors have the rotor on the outside.  As a result, the inrunners spins with a relatively lightweight rotor, which means inrunner motors typically spin fast but have low torque (i.e. inrunner motors have high KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). In order to obtain useful torque from the motor, we need to design and install a separate gearbox to convert the mechanical energies\cite{invsoutrunner}. The outrunner has a more massive rotor and so the spin-up time due to the additional inertia, but we can consider this as a benefit: the rotating mass is further from the center of rotation, thus the angular momentum helps to stabilize the rotor and reduce vibrations --- which is a major loss of efficiency. For an MRPAS typically mainly operating in stabilized flight and hover, this ``compromise'' is acceptable. Lastly, outrunners technologies are more modern and more serviceable than inrunner motors\cite{invsoutrunner}. For the reasons above, we choose outrunner motors.
```

\paragraph{Motor Size and Speed}\label{section:motor-speed}

Commercial outrunner BLDC motors generally have the following design parameters: rotor size, KV or KT constants, voltage, maximum rated power, and weight. Please refer to Appendix~\ref{appendix:droneengine} for the relationship between the aforementioned design parameters. Our design process consists of screening and ranking the best motors using the following order:

\begin{enumerate}
    \item Voltage: we choose motor that operates between 12.0 V to 20.0 V which is enough for heavy operations.

>     \item KV constant: Since the motors' RPM-per-volt ratio (KV constant) is inversely proportional of torque-per-current ratio (KT constant), manufacturers typically only focus on the KV constant. We choose motors that has KV constants between 500 KV to 1000 KV. These constants are considered low-speed and high-torque characteristics and are typically used for large drones and endurance flights. We also choose this range so the propellers can spin slower and more efficiently, which extends the maximum flight time.

Reviewer 2: ```suggestion
    \item KV constant: Since the motors' RPM-per-volt ratio (KV constant) is inversely proportional to the torque-per-current ratio (KT constant), manufacturers typically only focus on the KV constant. We choose motors that have KV constants between 500 KV to 1000 KV. These constants are considered low-speed and high-torque characteristics and are typically used for large drones and endurance flights. We also choose this range so the propellers can spin slower and more efficiently, which extends the maximum flight time.
```

\paragraph{Motor Type}

\underline{\textit{Brushed or brushless?}}
The MRPAS is electrically powered by a DC battery and we have two options for motors: DC brushed motors and DC brushless motors. Brushed motors use mechanical brushes in contact with the rotor’s commutator to switch polarity to sustain a constant rotation, whereas the brushless motors use electronic controllers to switch the polarity in the rotor. 

For flight applications, we chose the brushless DC (BLDC) motors for their extended usable lifetime. BLDC motors are also much more efficient than brushed motors because as they do not mechanically swap the polarity in the motor, which results in excess friction, heat, and potential sparking. 

\underline{\textit{Inrunner or outrunners?}}

> BLDC motors are further broken down into two types: \textit{inrunner} and \textit{outrunner}. Inrunner motors have the rotor on the inside and the outrunner motors have the rotor on the outside.  As a result, the inrunners spins with a relatively lightweight rotor, which means inrunner motors typically spin fast but have low torque (i.e. inrunner motors have high KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). In order to obtain useful torque from the motor, we need to design and install a separate gearbox to convert the mechanical energies\cite{invsoutrunner}. The outrunner has a more massive rotor and so the spin-up time due to the additional inertia, but we can consider this as a benefit: the rotating mass is further from the center of rotation, thus the angular momentum helps to stabilize the rotor and reduce vibrations --- which is a major loss of efficiency. For an MRPAS typically mainly operating in stabilized flight and hover, this ``compromise'' is acceptable. Lastly, outrunners technologies are more modern and more serviceable than inrunner motors\cite{invsoutrunner}. For the reasons above, we choose outrunner motors.

Reviewer: ```suggestion
BLDC motors are further broken down into two types: \textit{inrunner} and \textit{outrunner}. Inrunner motors have the rotor on the inside and the outrunner motors have the rotor on the outside. Inrunners spin with a relatively lightweight rotor, implying that they typically spin fast but have low torque (i.e. inrunner motors have high KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). In order to obtain useful torque from the motor, we would need to design and install a separate gearbox to convert the mechanical energy\cite{invsoutrunner}. Outrunners have a more massive rotor and thus a larger spin-up time due to the additional inertia but we can consider this as a benefit: the rotating mass is further from the center of rotation, thus the angular momentum helps to stabilize the rotor and reduce vibrations --- which is typically a major loss of efficiency. For an MRPAS typically operating in stabilized flight and hover, this ``compromise'' is acceptable. Finally, outrunners are more serviceable than inrunner motors\cite{invsoutrunner}. For the reasons above, we chose outrunner motors.
```

Reviewer: My last comment on this section seems to have disappeared

\paragraph{Speed Controllers}

> The electronic speed controller (ESC) controls the voltage, current, and phase supplied to a BLDC motor. Since BLDC motors function by rotating magnetic field using electric commutation, the ESCs need to precisely control the timing and phase of the input to the three phase lines. All ESCs function similarly and the design parameters are size/weight, power rating, power efficiency, and cost.

Reviewer: ```suggestion
The electronic speed controller (ESC) controls the voltage, current, and phase supplied to a BLDC motor. Since BLDC motors function by rotating a magnetic field through electric commutation, the ESCs need to precisely control the timing and phase of the input to the three phase lines. All ESCs function similarly, with the pertinent design parameters being size/weight, power rating, power efficiency, and cost.
```
"Phase lines"?

    \end{tabular} 
\end{table}

\subsubsection{Flight Controller Units}

> Flight controllers units (FCU) or sometimes short as FC come in different tiers differentiated by their price and target application/market. 

Reviewer: ```suggestion
Flight controller units (FCUs)  come in different tiers differentiated by their price and target application/market. 
```

> The cheapest FCU typically only has the bare-minimum features for flight (such as accelerometers and self-hover functions). These flight controllers are typically designed for acrobatic or basic VLOS flying and typically cost CA\$20 to CA\$50. The firmware that runs on these FCUs are typically open-source but nonetheless user-friendly and easily-programmable. Below is a list of FCs which falls under this tier:

Reviewer: ```suggestion
The cheapest FCUs typically only have the bare-minimum features for flight (such as accelerometers and self-hover functions). These flight controllers are typically designed for acrobatic or basic VLOS flying and typically cost CA\$20 to CA\$50. The firmware that runs on these FCUs is typically open-source and rudimentary but nonetheless user-friendly and easily-programmable. Below is a list of FCs which falls under this tier:
```

Define VLOS

    \item CC3D FC is reliable with auto-tuning PIDs.
\end{itemize}

> The next tier of FCs have more advanced features such as GPS-hold, autonomous flight, anto-land, telemetry, etc. They are designed for more advanced operation. These FCs are more expensive, costing CA\$50 to CA\$200, however these FCs have excellent after-sale support for their respective 

Reviewer: ```suggestion
The next tier of FCs include more advanced features such as GPS-hold, autonomous flight, auto-land, telemetry, etc. They are designed for more advanced operations. These FCs are more expensive, costing CA\$50 to CA\$200, however these FCs have excellent after-sale support as their respective 
```

> manufacturers and their documentation is better. Along 

Reviewer: ```suggestion
manufacturers and documentation is better. Along 
```

\textit{Configuration} is how the Li-Po batteries are manufactured and wired together: a single cell in a Li-Po battery provide a 
nominal voltage of 3.7 V due to its electrochemistry.
Similar to how portable electronics and appliances require AA or AAA batteries placed in a certain order, a single cell of 3.7 V is not enough to drive anything powerful. Ergo multiple cells can be wired in series and the 
voltage adds up: 2 cells in series, denoted as``2S'' batteries, have 7.4 V output, 3S batteries have 11.1 V output, and 4S batteries have a 14.8 V output. 

> Battery \textit{capacity} is measured in watt-hours (Wh) or milli-amp-hours (mAh), which are units of electric power multiplied by units of time. A 20.0 Wh battery can output 20.0 W for 1.0 hour. Multiple battery cells can be put in parallel and their capacity adds up. A pair of four-cell batteries wired in parallel, denoted by ``4S2P'' have a 14.8 V output while having double the capacity of a single 4S battery of the same variant.

Reviewer 3: ```suggestion
Battery \textit{capacity} is measured in watt-hours (Wh) or milli-amp-hours (mAh), which are units of electric power multiplied by units of time. A 20.0 Wh battery can output 20.0 W for 1.0 hour. Multiple battery cells can be put in parallel and their capacity adds up. A pair of four-cell batteries wired in parallel, denoted by ``4S2P'' has a 14.8 V output while having double the capacity of a single 4S battery of the same variant.
```

    \item DJI Naza M Lite.
\end{itemize}

> For the applications we require, we chose the ArduPilot APM 2.8 or the Pixhawk PX 4 as it is the most flexible option with advanced features at a reasonable price of CA\$75.00. Additionally, due to of the low-cost of low-tier FCs, we will also purchase one of the Naze32 or CC3D FC modules as a back-up.

Reviewer: ```suggestion
For the applications we require, we chose the ArduPilot APM 2.8 or the Pixhawk PX 4, as they are the most flexible option with advanced features at a reasonable price of CA\$75.00. Additionally, due to of the low-cost of low-tier FCs, we will also purchase one of the Naze32 or CC3D FC modules as a back-up.
```

\subsubsection{Battery}

> We chose lithium polymer (Li-Po) batteries as they have the highest energy density (highest capacity to weight ratio) similar to that of lithium-ion (Li-ion) batteries, and thus are perfect for high-power, low-weight applications. Figure~\ref{fig:batterytypes} shows a graph of energy density of types of batteries \cite{battery} --- from this graph, it is clear that Li-Po batteries are to be chosen.

Reviewer: ```suggestion
We chose lithium polymer (Li-Po) batteries as they have the highest energy density (highest capacity to weight ratio), similar to that of lithium-ion (Li-ion) batteries, and are thus perfect for high-power, low-weight applications. Figure~\ref{fig:batterytypes} shows a the energy densities of different types of batteries \cite{battery} --- from this graph, it is clear that Li-Po batteries are to be chosen.
```

\begin{figure}[h]
    \centering
    \includegraphics[scale=0.5]{img/energydensity.png}
    \caption{Energy Density Graph of Various Battery Types}
    \label{fig:batterytypes}
\end{figure}

Li-Po batteries have mainly three design parameters: configuration, capacity,  discharge rating, and internal resistance.

> \textit{Configuration} is how the Li-Po batteries are manufactured and wired together: a single cell in a Li-Po battery provide a 

Reviewer: ```suggestion
\textit{Configuration} is how the Li-Po batteries are manufactured and wired together: a single cell in a Li-Po battery provides a 
```

nominal voltage of 3.7 V due to its electrochemistry.
Similar to how portable electronics and appliances require AA or AAA batteries placed in a certain order, a single cell of 3.7 V is not enough to drive anything powerful. Ergo multiple cells can be wired in series and the 

> voltage adds up: 2 cells in series, denoted by ``2S'' batteries have 7.4 V output, 3S batteries have 11.1 V output, and 4S batteries have 14.8 V output. 

Reviewer: ```suggestion
voltage adds up: 2 cells in series, denoted as``2S'' batteries, have 7.4 V output, 3S batteries have 11.1 V output, and 4S batteries have a 14.8 V output. 
```

> Battery \textit{capacity} is measured in watt-hours (Wh) or milli-amp-hours (mAh), which is unit of electric power multiplied by unit time. A 20.0 Wh battery can output 20.0 W for 1.0 hour. Multiple battery cells can be put in parallel and their capacity adds up. A pair of four-cell batteries wired in parallel, denoted by ``4S2P'' have a 14.8 V output while double the capacity of a single 4S battery of the same variant.

Reviewer: ```suggestion
Battery \textit{capacity} is measured in watt-hours (Wh) or milli-amp-hours (mAh), which are units of electric power multiplied by units of time. A 20.0 Wh battery can output 20.0 W for 1.0 hour. Multiple battery cells can be put in parallel and their capacity adds up. A pair of four-cell batteries wired in parallel, denoted by ``4S2P'' have a 14.8 V output while having double the capacity of a single 4S battery of the same variant.
```

> The \textit{discharge rating} is denoted by ``C'' and indicates the ratio of he capacity and maximum discharge current safely without significantly harming the battery's life. A 1,000 

Reviewer: ```suggestion
The \textit{discharge rating} is denoted by ``C'' and indicates the ratio of the capacity and the maximum current which can be safely discharged without significantly harming the battery's life. A 1,000 
```

> mAh battery with a discharge rating of 20 C can discharge a maximum of 1.0 A $\times$ 20 C = 20 A. It is important to note that a high discharge rating does not imply improved performance, but the \textit{internal resistance}. Since a higher internal resistance means that the battery can run less efficiently with power lost internally\cite{battery-c}. In reality, the internal resistance is merely a value on the data-sheet and not a design parameter --- thus we will choose the battery with the most optimal internal resistance as alotted by our budget. 

Reviewer: ```suggestion
mAh battery with a discharge rating of 20 C can discharge a maximum of 1.0 A $\times$ 20 C = 20 A. It is important to note that a high discharge rating does not imply improved performance due to its \textit{internal resistance}. A higher internal resistance implies that the battery can run less efficiently, as power is lost internally\cite{battery-c}. We will thus choose the battery with the most optimal internal resistance as allotted by our budget. 
```
